x-pack/test/apm_api_integration/tests/sourcemaps/sourcemaps.ts

```diff
@@ -139,7 +153,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
           sourcemap: SAMPLE_SOURCEMAP,
         });
 
-        await waitForSourceMapCount(1);
+        await waitForSourceMapCount(1); // TODO: rename to "waitForSourceMapInApmIndexCount"
```

Comment:
what's holding you back from changing this?

Reply:
Nothing - just forgot.